components/home/mcp-card.tsx

```
@@ -17,18 +17,22 @@ interface McpCardProps {
 }
 
 // Recursively inject envVars into localConfig where referenced by key
-function injectEnvVarsToLocalConfig(localConfig: any, envVars: any[] = [], apiKey: string | undefined) {
+function injectEnvVarsToLocalConfig(
+    localConfig: any,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)